lib/web/apiserver.go

```diff
@@ -50,6 +50,7 @@ import (
 	"golang.org/x/crypto/ssh"
 	"golang.org/x/exp/slices"
 	"golang.org/x/mod/semver"
+	"golang.org/x/time/rate"
```

Comment:
Using this instead of oxy ratelimiter, as the latter is quite tightly coupled with individual `http.Request`-s.